src/main/java/Events/MathEvents.java

```diff
@@ -1,58 +1,50 @@
 package Events;
 
+import Handlers.MathHandlers;
+import net.dv8tion.jda.api.EmbedBuilder;
 import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
 import net.dv8tion.jda.api.hooks.ListenerAdapter;
 import javax.annotation.Nonnull;
+
 import static Bot.DiscordBot.PREFIX;
+import static java.awt.Color.GREEN;
+import static java.awt.Color.RED;
 
 public class MathEvents extends ListenerAdapter {
 
     @Override
     public void onGuildMessageReceived(@Nonnull GuildMessageReceivedEvent event) {
+
+        MathHandlers mathHandlers = new MathHandlers();
+
+        // Response card
+        EmbedBuilder iCard = new EmbedBuilder();
```

Comment:
You could put this as an instance variable instead but this is not critical, but it would be effective. if you'r gonna reuse it several times in your class